merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/MissionModelParser.java

```diff
@@ -581,6 +582,22 @@ private Optional<EffectModelRecord> getActivityEffectModel(final TypeElement act
         );
 
         parameterizedDuration = Optional.of(executableElement.getSimpleName().toString());
+      } else  if (element.getAnnotation(ActivityType.MaximumDuration.class) != null){
```

Comment:
Minor, nonblocking: extra space
```suggestion
      } else if (element.getAnnotation(ActivityType.MaximumDuration.class) != null){
```